lib/puppet/type/postgresql_psql.rb

```diff
@@ -119,6 +119,8 @@ def matches(value)
     newvalues(:true, :false)
   end
 
+  autorequire(:class) { 'postgresql::server::service' }
```

Comment:
I don't know if this actually works. I had to add some ```require => Class['postgresql::server::service']``` so it might not.

Reply:
Turns out it didn't but rspec-puppet was able to test for it. Now updated so it does work.